docs/articles/documentation/reference/test-api/role/constructor.md

```diff
@@ -8,7 +8,7 @@ permalink: /documentation/reference/test-api/role/constructor.html
 Creates and initializes a [user role](../../../guides/advanced-guides/authentication.md#user-roles).
 
 ```text
-Role( url, func( t ) [, options] )
+Role(url, func(t) [, options])
```

Comment:
Should not we specify a return value here? See [ClientFunction constructor](https://github.com/DevExpress/testcafe/pull/5677/files#diff-73631fdf56905daa922a371dcd130a0a998eee2d770516356dddbd48f55bf21aR11)

Reply:
> Role(url, func(t) [, options]) → Role

[Source](https://github.com/DevExpress/testcafe/blob/afb1e857987cb2623c23529237716b34fb42bb5c/ts-defs-src/test-api/role.d.ts#L24)